datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -238,6 +238,22 @@ def get_instance_proxy(self, instance, uri, proxies=None):
     def _context_uid(self, mtype, name, tags=None, hostname=None):
         return '{}-{}-{}-{}'.format(mtype, name, tags if tags is None else hash(frozenset(tags)), hostname)
 
+    def submit_histogram_bucket(self, check_id, name, value, lower_bound, upper_bound, monotonic, hostname, tags):
```

Comment:
Oups good catch probably a copy/paste of the aggregator signature.

The `monotonic` parameter is used to determine if we need to store the bucket previous value in the aggregator to compute the delta (see https://github.com/DataDog/datadog-agent/pull/3937/commits/1b5b8725702c24c48ba247063ce4a46c9092e6ce) this would always be true for prometheus/openmetrics for now but this is giving us flexibility in case of other/custom histograms.